app/auth/views.py

```diff
@@ -95,3 +117,33 @@ def login():
 		#an error occured in the server
 		response = {'message': str(e)}
 		return jsonify(response), 500
+
+@auth.route('/gettoken', methods=['post'])
```

Comment:
Do you need to have a URL route for this method? should it be a stand-alone method?

Reply:
this is what will first be called when a user requests for password reset so that a confirmation email will be sent to them.